### PR TITLE
[candi] Ubuntu 22.04 support

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

Ubuntu 22.04 support.
## Why do we need it, and what problem does it solve?

Added new ubuntu LTS version support
## Changelog entries

```changes
section: candi
type: feature
summary: Added Ubuntu 22.04 LTS support
```

### CODE REVIEW

**Review by yalosev — DISMISSED** [review]

**Review by name212 — APPROVED** [review]

File: candi/version_map.yml [hunk 1]
```diff
@@ -60,6 +60,29 @@ bashible: &bashible
...
+        containerd:
+          desiredVersion: "containerd.io=1.5.11-1"
+          allowedPattern: "containerd.io=1.[56]"
+      containerd:
+        desiredVersion: "containerd.io=1.5.11-1"
+        allowedPattern: "containerd.io=1.[56]"
+      kernel:
+        generic:
+          desiredVersion: "5.15.0-27-generic"
+          allowedPattern: ""
```
Comment (konstantin-axenov) [inline candi/version_map.yml:76]: Why not to add `allowedPattern: "5.15"` to allow minor patches to kernel? @nabokihms What do you think?

Comment (konstantin-axenov) [inline candi/version_map.yml:76]: Why not to add `allowedPattern: "5.15"` to allow minor patches to kernel? @nabokihms What do you think?

Comment (konstantin-axenov) [reply candi/version_map.yml:76]: Maksim approved, let's do it.

Comment (RomanenkoDenys) [reply candi/version_map.yml:76]: ok

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**